File: 6kyu/vin_checker.py/example_test_cases.py
import codewars_test as test


@test.describe("Tests")
def tests():
    @test.it("Fixed Tests")
    def fixed_tests():
        test.assert_equals(check_vin("5YJ3E1EA7HF000337"), True)
        test.assert_equals(check_vin("5YJ3E1EAXHF000347"), True)
        test.assert_equals(check_vin("5VGYMVUX7JV764512"), True)
        test.assert_equals(check_vin("7WDMMTDV9TG739741"), False)
        test.assert_equals(check_vin("7JTRH08L5EJ234829"), False)

File: 6kyu/vin_checker.py/javascript/example_test_cases.js
const chai = require("chai");
const assert = chai.assert;

describe("Tests", function () {
  it("Fixed Tests", function () {
    assert.strictEqual(checkVin("5YJ3E1EA7HF000337"), true);
    assert.strictEqual(checkVin("5YJ3E1EAXHF000347"), true);
    assert.strictEqual(checkVin("5VGYMVUX7JV764512"), true);
    assert.strictEqual(checkVin("7WDMMTDV9TG739741"), false);
    assert.strictEqual(checkVin("7JTRH08L5EJ234829"), false);
  });
});
